test/test.jl

```diff
@@ -24,6 +24,7 @@
         @test_nolog(logger, "info", msg, Memento.debug(logger, msg))
         @test_nolog(logger, "info", msg, Memento.info(logger, different_msg))
         @test_nolog(logger, "info", msg, Memento.warn(logger, different_msg))
+        @test_nolog(logger, "info", r".*foo.*", info(logger, "baz"))
```

Comment:
```suggestion
        @test_nolog(logger, "info", r".*foo.*", Memento.info(logger, "baz"))
```

Fixes tests in 0.7

Should drop 0.7 but that is another PR